packages/ble-transport/README.md

```diff
@@ -0,0 +1,29 @@
+<p align="center">
```

Comment:
Really nit, but I would maybe call it `transport-ble` instead of `ble-transport` so we have category-specific. E.g. eslint also have `eslint-plugin-import`, `eslint-plugin-prettier` instead of `prettier-plugin-eslint` & `import-plugin-eslint`. 

It means you can easily group the dependencies by transport (as package names are sorted).

Reply:
Makes sense. On it